src/node/mod.rs

```diff
@@ -1037,9 +1037,10 @@ impl Node {
     }
 
     /// Returns the age counter of the given node if it is member of the same section as this node,
-    /// `None` otherwise.
+    /// and this node is elder. Returns `None` otherwise.
```

Comment:
`if it is member of the same section as this node, and this node is elder.`  => `if it is an elder of the same section as own.`
btw. I think then better to update the function name to reflect this scope change?

Reply:
OK, that might be awkwardly worded, but what it tried to say is that if **we** are elder, this returns the age counter of that node, regardless of whether they are elder not not, otherwise it returns `None`. I'll try to reword it.